server/test/auth.spec.js

```diff
@@ -0,0 +1,19 @@
+const chai = require("chai");
+const chaiHttp = require("chai-http");
+const { server } = require("../app.js");
+
+chai.should();
+chai.use(chaiHttp);
+
+describe("GET /auth/logout", () => {
+  it("should return 200 and a response message", (done) => {
+    chai
+      .request(server)
+      .get(`/auth/logout/`)
+      .end((err, res) => {
+        res.should.have.status(200);
+        res.text.should.equal("You have successfully logged out");
+        done();
+      });
+  });
+});
```

Comment:
Just so we are aware, `GET` isn't the verb that makes the most sense for a `/logout` path! (see [this article](https://www.baeldung.com/logout-get-vs-post) for more details on why) For now though I think this is fine to leave as-is 👍🏼 

The existing implementation may have thrown you off but this is the pitfall of developers building tests - we have to be careful not to be biased by our knowledge of the implementation and focus on specs!